.gitignore

```diff
@@ -14,3 +14,6 @@ cypress/videos/*
 .vscode/*
 !.vscode/extensions.json
 !.vscode/settings.json
+
+# OS files
+.DS_Store
```

Comment:
This should be done in your global `.gitignore` file instead of part of this repo, because its part of the environment you have.